chapter_convolutional-neural-networks/channels.md

```diff
@@ -105,19 +105,19 @@ corr2d_multi_in(X, K)
 ```{.python .input}
 #@tab pytorch
 X = torch.tensor([[[0, 1, 2], [3, 4, 5], [6, 7, 8]],
-                  [[1, 2, 3], [4, 5, 6], [7, 8, 9]]])
+                  [[1, 2, 3], [4, 5, 6], [7, 8, 9]]], dtype=torch.float32)
```

Comment:
why not just use `torch.tensor([[[0.0, 1,...`, namely add `.0` for the first element?

Reply:
Yes. We can do that and the tensor would automatically infer the dtype. But it is always better to explicitly give dtype as an argument, which ensures that the code is more interpretable to the reader.

Also, a reader may wonder why we change only the first element to 1.0 or why we do not do such a thing in mxnet.
Thus I'd suggest explicitly mentioning dtypes wherever possible, but we can surely skip this if the input data is clearly float or int etc.